rules/windows/process_creation/proc_creation_win_net_use_mount_share.yml

```
@@ -0,0 +1,29 @@
+title: Windows Share Mounted Via Net.EXE
+id: f117933c-980c-4f78-b384-e3d838111165
+related:
+    - id: 3abd6094-7027-475f-9630-8ab9be7b9725
+      type: similar
+status: experimental
+description: Detects when a share is mounted using the "net.exe" utility
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+author: Nasreddine Bencherchali (Nextron Systems)
+date: 2023/02/02
+tags:
+    - attack.lateral_movement
+    - attack.t1021.002
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        Image|endswith:
+            - '\net.exe'
+            - '\net1.exe'
+        CommandLine|contains:
+            - ' use '
+            - ' \\'
```

```suggestion
            - ' \\\\'
```
`\\\\` if we want to match a literal `\\`

---

Both are correct ways of writing. The `\` is not a character that needs to be escaped in SIGMA.

---

ah yes you are correct. I was coming from the rule with wildcards and my mind was preoccupied.

---

on 2nd thought: My first statement was true. `\\` will be escaped to `\` --> we need `\\\\` to get a literal `\\`

---

approved after applying this